### DISCUSSION TITLE
Change port in config files?

### DISCUSSION THREAD

**Discussion by mattjt21** [Geektoolkit/Dynaframe3#160] [General]
Good evening, 

I am already running DarkIce on port 8000.  I installed Dynaframe but can't get to it.  I can't figure out which file has the config setting for the default port.  Can anyone help me out?

Thank you!

**Comment by mattjt21** [comment]
I updated to 2.24 and found the AppSettings.cs. I didn't have it in the directory for the earlier version.
Now to just get it to run..

**Comment by Geektoolkit** [comment]
hey sorry, just seeing this. If 8000 is used, it may default to 4200 or another backup port.  If you edit appsettings.cs though and build that'd likely be a good way to go. Also if you run it once and then look at the appsetting.json you may try to edit the port there. 
  I'm working on a new version, but the new one is designed to run more standalone, and uses port 80. For those that want to run other stuff alongside, Dynaframe 2 is the way to go

**Comment by drinehimer** [comment]
I am also in the processing of some major changes (split "server" so it can run in docker, another pi/server, or still on the same pi). As part of this I am planning to make it very easy to set the port you want. It will take me some time to get this finished up though.
